examples/stable/cassandra/README.md

```diff
@@ -28,9 +28,12 @@ To install the cassandra in your Kubernetes cluster you can run below command
 $ helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com
 $ helm repo update
 # remove app-namespace with the namespace you want to deploy the cassandra app in
-$ helm install --namespace "<app-namespace>" "cassandra" incubator/cassandra --set image.repo=kanisterio/cassandra --set image.tag=0.40.0 --set config.cluster_size=2
+$ kubectl create ns <app-namespace>
+$ helm install cassandra bitnami/cassandra --namespace <app-namespace> --set image.repository=kanisterio/cassandra --set image.tag=0.40.0 --set cluster.replicaCount=2 
+
+
 ```
-This command will install cassandra on your Kubernetes cluster with 2 nodes. You can notice that we are using custom image of cassandra in the helm to install the cassandra cluster. The reason is we have to use some Kanister tools to take backup, so only change that we have done is including that tooling on top of statndard `cassandra:3.11.3` image.
+This command will install cassandra on your Kubernetes cluster with 2 nodes. You can notice that we are using custom image of cassandra in the helm to install the cassandra cluster. The reason is we have to use some Kanister tools to take backup, so only change that we have done is including that tooling on top of statndard `cassandra:3.11.8-debian-10-r20` image.
```

Comment:
Let's change every occurrence of `cassandra` to `Cassandra`, we ll not every but where we mean Cassandra as db.